robottelo/performance/perf_concurrent_test.py

```diff
@@ -0,0 +1,423 @@
+"""Test utilities for writing performance tests
+
+Define ConcurrentTestCase as parent class of writting Subscribe by AK,
+Subscribe by register and attach, Subscription Deletion.
+
+"""
+import csv
+
+from robottelo.common import conf, ssh
+from robottelo.test import TestCase
+
+from robottelo.performance.perf_thread import DeletePerfThread
```

Comment:
The following is better than repeating the same path when using `from ... import ...`
```python
from robottelo.performance.perf_thread import (
    DeletePerfThread,
    SubscribeAKPerfThread,
    SubscribeAttachPerfThread,
)
```